pkg/apis/core/v1alpha1/federatedconfig_types_test.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
please do not commit the api test files.

Reply:
It was deleted. I will merge 2 commits into 1 to make it clear: the deletion files do not need to be committed at first time.